Sources/SwiftFusion/Optimizers/CGLS.swift

```diff
@@ -32,30 +32,31 @@ public class CGLS {
   /// Optimize the Gaussian Factor Graph with a initial estimate
   /// Reference: Bjorck96book_numerical-methods-for-least-squares-problems
   /// Page 289, Algorithm 7.4.1
-  public func optimize(gfg: GaussianFactorGraph, initial: inout VectorValues) {
+  public func optimize<Objective: MatrixLinearLeastSquaresObjective>(
```

Comment:
So objective is inferred automatically? That’s nice!

Reply:
yep